src/pages/LHStoryPage.js

```diff
@@ -16,6 +17,21 @@ const customStyles = {
   }
 }
 
+const mapStateToPropsWidget = (state) => ({
+  transactions: state.get('lhStory').transactions,
+  toBlock: state.get('lhStory').toBlock,
+  isFetching: state.get('lhStory').isFetching,
+  title: 'LHT Transactions from all accounts'
+})
+
+const mapDispatchToPropsWidget = (dispatch) => ({
+  getTransactions: (toBlock = null, isAllLH = false) => {
```

Comment:
isAllLH passed and not used. we don't need it here at all if we use different action-creator...